.golangci.yml

```diff
@@ -1,29 +1,42 @@
+version: "2"
 run:
-  timeout: 3m
```

Comment:
I think we should keep the timeout. Or is it there by default?

Reply:
Probably bug in migration tool. Reinstated. Thanks!